src/main/java/org/craftercms/studio/controller/rest/v2/ExceptionHandlers.java

```diff
@@ -203,6 +204,14 @@ public ResponseBody handleCmisPathNotFoundException(HttpServletRequest request,
         return handleExceptionInternal(request, e, response);
     }
 
+    @ExceptionHandler(PasswordRequirementsFailedException.class)
+    @ResponseStatus(HttpStatus.INTERNAL_SERVER_ERROR)
+    public ResponseBody handlePasswordRequirementsFailedException(HttpServletRequest request,
+                                                                  PasswordRequirementsFailedException e) {
+        ApiResponse response = new ApiResponse(ApiResponse.INTERNAL_SYSTEM_FAILURE);
```

Comment:
It's not an internal system failure, it's a bad param. Make sure to include a more detailed version of the error in the response codes so the API know what's wrong, and remediation.